server/tserver/src/main/java/org/apache/accumulo/tserver/memory/LargestFirstMemoryManager.java

```diff
@@ -56,6 +60,8 @@ public class LargestFirstMemoryManager {
   private double compactionThreshold;
   private long maxObserved;
   private final HashMap<TableId,Long> mincIdleThresholds = new HashMap<>();
+  private final Cache<TableId,Long> mincAgeThresholds =
+      Caffeine.newBuilder().expireAfterWrite(5, TimeUnit.MINUTES).build();
```

Comment:
The [Caches](https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/util/cache/Caches.java) class exists in 3.1 that serves as a central point for creating Caffeine caches. I'm wondering if the `mincIdleThresholds` should be a Cache as well.

Reply:
ok, I can update to use that

Reply:
I could only find the Caches class i main branch.  I updated the code to use it in this merge commit ab996d0d26df4b2ed2c1874289d5b785ecbe2686

Reply:
>  I'm wondering if the mincIdleThresholds should be a Cache as well.

yeah it should be, I will create a 2.1 PR for that

Reply:
Attempted to make the change in 2.1 and found the code was clearing map on each pass through all tablets.  So changed direction and opened #4855